lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -3743,6 +3751,10 @@ class ByteCodeBufferReader
             (*functionBody)->m_isAsmjsMode = (bitflags & ffIsAsmJsMode) ? true : false;
 #endif
 
+            if ((*functionBody)->paramScopeSlotArraySize > 0)
```

Comment:
Yeah, this commit somehow swallowed Aneesh's when I rebased on his change. Only my own commit is relevant.

-- Paul
________________________________
From: Akrosh Gandhi<mailto:notifications@github.com>
Sent: ‎3/‎29/‎2016 9:43 AM
To: Microsoft/ChakraCore<mailto:ChakraCore@noreply.github.com>
Cc: Paul Leathers<mailto:pleath@microsoft.com>
Subject: Re: [Microsoft/ChakraCore] Deserializing anonymous function (#674)


In lib/Runtime/ByteCode/ByteCodeSerializer.cpp<https://github.com/Microsoft/ChakraCore/pull/674#discussion_r57756636>:

> @@ -3743,6 +3751,10 @@ class ByteCodeBufferReader
>              (*functionBody)->m_isAsmjsMode = (bitflags & ffIsAsmJsMode) ? true : false;
>  #endif
>
> +            if ((*functionBody)->paramScopeSlotArraySize > 0)


@aneeshdk<https://github.com/aneeshdk> already fixed this.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/674/files/8c01d7b9880fcb7a5f6816a3152362bda85b9df7#r57756636>